src/deviceApiCalls/transports/apple.transport.js

```diff
@@ -0,0 +1,52 @@
+import {MissingWebkitHandler, wkSendAndWait} from '../../appleDeviceUtils/appleDeviceUtils'
+import {DeviceApiTransport} from '../../../packages/device-api'
+import {GetRuntimeConfigurationCall} from '../__generated__/deviceApiCalls'
+
+export class AppleTransport extends DeviceApiTransport {
+    /** @type {GlobalConfig} */
+    config
+
+    /** @param {GlobalConfig} globalConfig */
+    constructor (globalConfig) {
+        super()
+        this.config = globalConfig
+    }
+
+    async send (rpc) {
+        try {
+            return await wkSendAndWait(rpc.method, rpc.params || undefined, {
```

Comment:
Previously, some calls were fire-and-forget and didn't need to wait for a response (`weSend`). Did we change that?

Reply:
Mhm, that's not entirely correct. On older devices we create methods that listen for a response (see `generateRandomMethod`). In this case, we would leave these methods there indefinitely, one for every fire-and-forget call. Is there a way to infer from the schema whether the call expects a response and if so short-circuit to `wkSend`?

Reply:
yep, we can do that 👍🏻